packages/core/sample-projects/kuai.config.ts

```diff
@@ -2,6 +2,10 @@ import { KuaiConfig } from '@ckb-js/kuai-core';
 
 const config: KuaiConfig = {
   port: 3000,
+  ckbChain: {
```

Comment:
`ckbChain` will be obtained through the `network` and `networks` values, so it is more expected to set these two instead of setting `ckbChain` directly, in addition there are some built-in network config, here can set it to `{ network: 'devnet' }` to achieve the same goal

Reply:
From https://github.com/ckb-js/kuai/blob/develop/packages/core/src/type/runtime.ts#L15, the `ckbChain` is necessary, maybe we can set it as optional?

If I don't define `ckbChain`, it will throw an exception when running `yarn dev` or `yarn build`.

Reply:
Any update? @PainterPuppets